recipes/libsrtp/all/conanfile.py

```diff
@@ -48,43 +53,85 @@ def requirements(self):
         if self.options.with_openssl:
             self.requires("openssl/[>=1.1 <4]")
 
+        if self.options.with_nss:
+            self.requires("nss3")
+
+        if self.options.with_mbedtls:
+            self.requires("mbedtls")
```

Comment:
This is missed a pinned version for the packages

Reply:
Added required versions